src/commands/show.ts

```diff
@@ -111,7 +117,10 @@ async function showProblemInternal(node: IProblem): Promise<void> {
 
         const originFilePath: string = await leetCodeExecutor.showProblem(node, language, outDir);
         const filePath: string = wsl.useWsl() ? await wsl.toWinPath(originFilePath) : originFilePath;
-        await vscode.window.showTextDocument(vscode.Uri.file(filePath), { preview: false, viewColumn: vscode.ViewColumn.One });
+        await Promise.all([
+            vscode.window.showTextDocument(vscode.Uri.file(filePath), { preview: false, viewColumn: vscode.ViewColumn.One }),
+            leetCodeConfig.get<boolean>("enableSideMode", true) ? previewProblem(node, true) : Promise.resolve(),
```

Comment:
Why we do nothing when `enableSideMode` is false.

Reply:
When opening a preview, clicking `code now`, the preview and text document will be both in Column 1 if `enableSideMode` is set to false. 

If we do `previewProblem` here, the result demo:
![demo](https://user-images.githubusercontent.com/20227484/56482743-59a28180-64f8-11e9-9e63-e883bc4e0220.gif)

Reply:
Oh, I got your point.

What to do when sideMode is enabled is:

1. Move the preview to 2nd column
2. show the source code file

What to do when sideMode is disabled is:
1. show the source code file

The method name `previewProblem()` is confusing here. I think we should wrap it with a method, named like `previewOnTheSideIfNeeded()`

Reply:
Updated (`movePreviewAsideIfNeeded`)